admin_manual/installation/nginx_configuration.rst

```diff
@@ -11,6 +11,10 @@ Nginx Configuration
    **ssl_certificate_key** to suit your needs.
 -  Make sure your SSL certificates are readable by the server (see `Nginx HTTP 
    SSL Module documentation <http://wiki.nginx.org/HttpSslModule>`_).
+-  **add_header** statements are only taken from the current level and are not cascaded 
+   from or to a different level. All necessary add_header statements must be defined 
+   in each level needed. For better readability is is possible to move ``common`` add 
+   header statements into a seperate file and include that file where ever necessary.
```

Comment:
And "wherever" is one word in this context - e.g. http://www.macmillandictionary.com/dictionary/british/wherever-possible-necessary